components/Dialogs/src/MDCAlertController+Customize.h

```diff
@@ -14,14 +14,19 @@
 
 #import "MaterialDialogs.h"
 
+/**
+ The Customize extensions privde APIs that extend the default Material Dialog. Make sure to adjust
+ your custom theming when using these APIs, if you are concerned with conforming to the Material
+ spec (material.io).
+*/
 @interface MDCAlertController (Customize)
 
 /**
- An optional custom icon view above the title of the alert.
+ An optional custom icon view that's displayed above the title of the alert, enabling custom
+ implementations of the view above the title.
```

Comment:
The ", enabling custom implementations of the view above the title." part is implied and can be removed.